src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -145,8 +145,11 @@ export default class IntegrationRepos extends AsyncComponent {
         </DropdownButton>
       );
     }
-    let repositories = this.state.integrationRepos.repos;
-    let items = (repositories || []).map(repo => {
+    const repositories = new Set(this.state.itemList.map(i => i.name));
+    let repositoryOptions = this.state.integrationRepos.repos.filter(repo => {
+      return !(repositories.has(repo.name) || repositories.has(repo.identifier));
```

Comment:
So why can't we just use `repo.identifier` only here? That should be the value we store no?

Reply:
yeah I guess so for GH, GHE, and BB. VSTS stores the name but `repo.name` here for VSTS is the full `project/repo` name